modules/network-observability-deploy-network-policy.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-deploy-network-policy_{context}"]
+= Configuring an ingress network policy by using the FlowCollector custom resource
+You can configure the `FlowCollector` custom resource to deploy an ingress Network Policy for Network Observability by setting the `spec.NetworkPolicy.enable` specification to `true`. By default, the specification is `false`.
+
+Ensure that Network Observability components you have configured can communicate with components that are external to the namespace, which might include:
+
+   * Connection to Loki (as defined in `FlowCollector` `spec.loki`)
+   * Connection to Kafka (as defined in `FlowCollector` `spec.kafka`)
+   * Connection to any exporter (as defined in FlowCollector `spec.exporters`)
+   * If you are using Loki and including it in the policy target, connection to an external object storage (as defined in your `LokiStack` related secret)
+
+.Procedure
+. . In the web console, navigate to *Operators* -> *Installed Operators*.
```

Comment:
Per ISG: [Use navigate only when a more direct verb, such as click, select, browse, or go to, is not applicable](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n)
```suggestion
. . In the web console, go to the *Operators* -> *Installed Operators* page..
```